backend/src/ee/services/access-approval-request/access-approval-request-service.ts

```diff
@@ -106,7 +106,7 @@ export const accessApprovalRequestServiceFactory = ({
       projectId: project.id,
       actorAuthMethod,
       actorOrgId,
-      projectOperationType: ProjectOperationType.SecretManager
+      projectOperationType: ActionProjectType.SecretManager
```

Comment:
nit: you might wan to update the param too: `projectOperationType` => projectType?